app/hackathon_variables.py

```diff
@@ -93,3 +95,12 @@
 BLACKLIST_ENABLED = False
 
 SUPPORTED_RESUME_EXTENSIONS = []
+
+MENTOR_EXPIRES = False
+VOLUNTEER_EXPIRES = False
+
+APPLICATION_EXPIRATION_TYPES = {
+    'H': True,
+    'M': MENTOR_EXPIRES,
+    'V': VOLUNTEER_EXPIRES,
+}
```

Comment:
This is impossible to understand, add documentation or make it more verbose.

Reply:
Move this to settings.py instead pls

Reply:
Done